FileValidator.js

```diff
@@ -3,6 +3,9 @@ var dom_config = require('z-dom-config');
 function FileValidator($input, dataAttrName, callback) {
     var self = this;
 
+    // This boolean variable if equals true, then validate function passes callback functions the all validation code in one array
+    this.is_fetch_array = false;
```

Comment:
I only use snake_case for local referencies for modules.
Local variables should use camelCasing.
Also, the name itself doesn't tell you much, it should be `returnErrorsInArray` instead.

Also, the comment should be "If this boolean variable equals true, then the validation function returns all the possible error codes as an array.".
And don't forget to break the line after 80 characters.